### PR TITLE
Add more useful error message when page fails to render

### PR DESCRIPTION
**Pull request by jranks123**
## Why are you doing this?

Adds the exception message to the reported error message in the instance where a page fails to render.

### CODE REVIEW

File: support-frontend/assets/helpers/render.js [hunk 1]
```diff
@@ -19,10 +19,10 @@ const getElementOrBody = (id: ?string): HTMLElement => {
   return element;
 };
 
-const renderError = (id: ?string) => {
+const renderError = (e: any, id: ?string) => {
   const element = getElementOrBody(id);
 
-  logException(['Fatal error rendering a page', id ? `id:${id}` : null].join(' '));
+  logException(['Fatal error rendering a page.', id ? `id:${id}.` : null, 'Error string:', e].join(' '));
```
Comment (joelochlann) [inline support-frontend/assets/helpers/render.js]: not your changes, but can we just use string interpolation here?

Comment (jranks123) [reply support-frontend/assets/helpers/render.js]: sure thing

File: support-frontend/assets/helpers/render.js [hunk 2]
```diff
@@ -19,10 +19,10 @@ const getElementOrBody = (id: ?string): HTMLElement => {
   return element;
 };
 
-const renderError = (id: ?string) => {
+const renderError = (e: any, id: ?string) => {
   const element = getElementOrBody(id);
 
-  logException(['Fatal error rendering a page', id ? `id:${id}` : null].join(' '));
+  logException(`Fatal error rendering a page. ${id ? id : ""} Error string: ${e}`);
```
Comment (joelochlann) [inline support-frontend/assets/helpers/render.js]: needs to be `e.message`

File: support-frontend/assets/helpers/render.js [hunk 3]
```diff
@@ -19,10 +19,10 @@ const getElementOrBody = (id: ?string): HTMLElement => {
   return element;
 };
 
-const renderError = (id: ?string) => {
+const renderError = (e: any, id: ?string) => {
```
Comment (joelochlann) [inline support-frontend/assets/helpers/render.js]: I'd give this a type of `Error`

File: support-frontend/assets/helpers/render.js (hunk 2 above)
Comment (joelochlann) [reply support-frontend/assets/helpers/render.js]: and for good measure you could log `e.stack` too

**Pull request merged**